Makefile

```diff
@@ -242,8 +242,7 @@ run-ci-e2e-byoh-test:
 	hack/run-ci-e2e-test.sh -t basic -m 0
 
 .PHONY: run-ci-e2e-upgrade-test
-run-ci-e2e-upgrade-test:
-	hack/run-ci-e2e-test.sh -t upgrade
+run-ci-e2e-upgrade-test: ;
```

Comment:
Consider removing the corresponding job in the release repo https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.17.yaml#L248